### PR TITLE
Week 8 code review

### PR DESCRIPTION
**Pull request by werdlerk**
Hello,

I would really appreciate a code review for the code I've added for this week 8's assignments.

Thanks!

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -34,10 +34,15 @@ gem 'mini_magick'
 
 # Payment processing
```
Comment (brandoncc) [inline Gemfile:35]: Great idea, it is so easy to forget what each gem does!

File: app/services/create_payment.rb [hunk 2]
```diff
@@ -0,0 +1,9 @@
+class CreatePayment
+
+  def call(event)
+    charge = event.data.object
+    user = User.find_by(stripe_customer_id: charge.customer)
+    Payment.create(amount_cents: charge.amount, user: user, reference_id: charge.id) if user
```
Comment (brandoncc) [inline app/services/create_payment.rb]: If you are not wrapping the create with an `if`/`else` there is a decent chance you might want to know if it fails. To raise an exception, use `create!` instead.

File: brew_elasticsearch.txt [hunk 3]
```diff
@@ -0,0 +1,19 @@
+elasticsearch: stable 2.3.1, HEAD
```
Comment (brandoncc) [inline brew_elasticsearch.txt]: Are these brew files supposed to be in your repo?

File: config/initializers/stripe.rb [hunk 4]
```diff
@@ -4,3 +4,8 @@
 }
 
 Stripe.api_key = Rails.configuration.stripe[:secret_key]
+
+StripeEvent.configure do |events|
+  events.subscribe 'charge.succeeded', CreatePayment.new
```
Comment (brandoncc) [inline config/initializers/stripe.rb:9]: Very nice, I like how you created services to receive these calls. I was wondering where I would see those used.

File: Gemfile (hunk 1 above)
Comment (werdlerk) [inline Gemfile:35]: Exactly, especially with larger projects the Gemfile becomes a bizarre collection of gems you have no idea what each one does.

File: app/services/create_payment.rb (hunk 2 above)
Comment (werdlerk) [inline app/services/create_payment.rb]: Hm yes, that would be better.

File: brew_elasticsearch.txt (hunk 3 above)
Comment (werdlerk) [inline brew_elasticsearch.txt]: uh, oops. Indeed, never meant to be committed. 
Basically some output of brew info <package> so I know how to start the services.

File: config/initializers/stripe.rb (hunk 4 above)
Comment (werdlerk) [inline config/initializers/stripe.rb:9]: On second thought I think it's a bit overkill for these Stripe event handlers to be in the services directory. Maybe these two handlers should be moved to a StripeHandler module with inline classes handling the specific event type.
Indeed, I also like the code being separate from the Stripe initializer though.

**Pull request merged**